dataset/dataset_config.json

```diff
@@ -1,5 +1,6 @@
 {
   "apply_blacklist": true,
   "output_directory": "dataset",
-  "source_directory": "../../final_project_shared/sprites"
+  "source_directory": "../../final_project_shared/sprites",
+  "sprite_size": 100
```

Comment:
LGTM, the path is a bit specific, but it works. Also sprite_size where is that used?